### ISSUE TITLE
Physical flaws

### ISSUE THREAD

**Issue by thomas-keynexus** [prmakerspace/prms-door#4]
Physical parts of the device (mounting point, gears) don't stay attached or fit too loosely.  Having them come off causes issues with operability of the system, and not all members can easily replace/repair them.

A better solution would be a 3d printed mounting plate & gears with better adhesion and better tolerances.

**Comment by eladam14** [comment]
I agree with the idea of using a card to come and go, but feel the application should have the bugs ironed out before becoming the only way to come and go. Individual keys should be supplied to all members until that time.
